nvm_helper.sh

```diff
@@ -0,0 +1,419 @@
+#!/bin/bash
```

Comment:
I was manually executing it while I was developing it. This would be fixed in the final version. Again I was just looking for high level feedback on how this logic could be leveraged by nvm.